src/SFA.DAS.FindApprenticeshipTraining.Api.UnitTests/Models/WhenCastingToProviderBaseTypesFromMediatorType.cs

```diff
@@ -403,9 +403,56 @@ public void The_If_There_Are_Delivery_Modes_To_Filter_And_Returns_No_Delivery_Mo
             var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1, new List<DeliveryModeType>
             {
                 DeliveryModeType.DayRelease
-            });
+            }, new List<FeedbackRatingType>());
+
+            response.Should().BeNull();
+        }
+
+        [Test, AutoData]
+        public void Then_If_There_Are_Ratings_To_Filter_Then_Matches_On_Values(string sectorSubjectArea, GetProvidersListItem source)
+        {
+            source.FeedbackRatings = new List<GetFeedbackRatingItem>
+            {
+                new GetFeedbackRatingItem
+                {
+                    FeedbackName = "Good",
+                    FeedbackCount = 1,
+                },
+                new GetFeedbackRatingItem
+                {
+                    FeedbackName = "Excellent",
+                    FeedbackCount = 1,
+                }
+            };
+            
+            var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1,new List<DeliveryModeType>(), new List<FeedbackRatingType>{FeedbackRatingType.Poor});
 
             response.Should().BeNull();
+
+        }
+        
+        
+        [Test, AutoData]
+        public void Then_If_There_Are_Multiple_Ratings_To_Filter_Then_Matches_On_Values(string sectorSubjectArea, GetProvidersListItem source)
+        {
+            source.FeedbackRatings = new List<GetFeedbackRatingItem>
+            {
+                new GetFeedbackRatingItem
+                {
+                    FeedbackName = "Good",
+                    FeedbackCount = 1,
+                },
+                new GetFeedbackRatingItem
+                {
+                    FeedbackName = "Excellent",
+                    FeedbackCount = 1,
+                }
+            };
+            
+            var response = new GetTrainingCourseProviderListItem().Map(source, sectorSubjectArea,1,new List<DeliveryModeType>(), new List<FeedbackRatingType>{FeedbackRatingType.Poor,FeedbackRatingType.Excellent});
+
+            response.Should().NotBeNull();
```

Comment:
Can you modify this so that instead of it just not checking for null, it checks for what it actually should be